pom.xml

```diff
@@ -99,7 +99,7 @@
         <dependency>
             <groupId>net.java.dev.jets3t</groupId>
             <artifactId>jets3t</artifactId>
-            <version>0.7.1</version>
+            <version>0.9.3</version>
```

Comment:
Other people identified issues with upgrading jets3t lib. You may want to follow https://groups.google.com/forum/#!topic/secor-users/6TlNaKuPLwA